### PR TITLE
enable fedora 24 for testing

### PR DESCRIPTION
**Pull request by ddholstad99**
I have tested this with fedora 24 and it seems to be working.

### CODE REVIEW

**Comment by jlruizmlg** [comment]
Hi @ddholstad99 thanks your your pull, you are totally right, thanks for your contribution!

**Comment by jlruizmlg** [comment]
hi @ddholstad99 , i'm looking your pull request, and we have profiles for Fedora 23, 24 and 25, you can modify to include this versions?

Or if you like i can merge your pull request and add this version later.

https://github.com/wazuh/wazuh/tree/master/etc/templates/config/fedora

File: manifests/params.pp [hunk 1]
```diff
@@ -137,6 +137,16 @@
               }
               }
             }
+            'Fedora': {
+              if ( $::operatingsystemrelease =~ /^24.*/ ) {
```
Comment (jlruizmlg) [inline manifests/params.pp]: We can use something like:

```if ( $::operatingsystemrelease =~ /^(23|24|25).*/ ) {```

or

```
if ( $::operatingsystemrelease =~ /^23.*/ ) or ( $::operatingsystemrelease =~ /^24.*/ ) or ( $::operatingsystemrelease =~ /^25.*/ ) { 
```

**Comment by jlruizmlg** [comment]
Hi @ddholstad99 awesome! thanks for your contribution, let me test and merge!

**Comment by ddholstad99** [comment]
Fedora 23 and 25 are now supported.

**Comment by jlruizmlg** [comment]
@ddholstad99 I saw! thanks!

**Comment by jnarvaezp** [comment]
Hi, @ddholstad99 I just tried the code and works, I can confirm the code is correct.
thanks for your contribution!

**Pull request merged**